Android/app/src/main/java/fashiome/android/activities/ProductFormActivity.java

```diff
@@ -380,7 +380,7 @@ public void onConnected(Bundle dataBundle) {
         if (location != null) {
             Toast.makeText(this, "GPS location was found! Lat:"+ location.getLatitude() +" Long:"+location.getLongitude(), Toast.LENGTH_SHORT).show();
             LatLng latLng = new LatLng(location.getLatitude(), location.getLongitude());
-            product.setAddress(new Address(latLng));
+            product.setAddress(new Address(Item.getRandomLocation(latLng.latitude, latLng.longitude,5000)));
```

Comment:
@rakheegupta

You have your trial code in there, then you have code that you are using to debug your own stuff and then you are merging it directly to master. We are really upset with the way you are approaching things here.